packages/reputation-miner/ReputationMiner.js

```diff
@@ -1463,18 +1483,8 @@ class ReputationMiner {
     }
   }
 
-  // Gas price should be a hex string
-  async setGasPrice(_gasPrice){
-    if (!ethers.utils.isHexString(_gasPrice)){
-      throw new Error("Passed gas price was not a hex string")
-    }
-    const passedPrice = ethers.BigNumber.from(_gasPrice);
-    const minimumPrice = ethers.BigNumber.from("1100000000");
-    if (passedPrice.lt(minimumPrice)){
-      this.gasPrice = minimumPrice.toHexString();
-    } else {
-      this.gasPrice = _gasPrice;
-    }
+  async setFeeData(_feeData){
```

Comment:
It's certainly not `async`, that's an artifact from my refactoring. I left it as its own function as I don't think that `ReputationMinerClient` should be setting properties on `this._miner` directly (though we're not perfect on that front, examining the codebase).

Reply:
Fair, but I do think it could be all done in one line:

```
this._miner.setFeeData(getFeeData("average", this.chainId, this._adapter, this._miner.realProvider));

```

Reply:
I've made this its own function with syntax I'm happier with, and call it as a one-liner where it's used.